packages/cli/commands/webhook.ts

```diff
@@ -42,7 +49,7 @@ export class Webhook extends Command {
 	 * get removed.
 	 */
 	static async stopProcess() {
-		console.log(`\nStopping n8n...`);
+		LoggerProxy.getInstance().info(`\nStopping n8n...`);
```

Comment:
Can we do it like this? (whole file)

```suggestion
		LoggerProxy.info(`\nStopping n8n...`);
```